### DISCUSSION TITLE
Finding Group after reimport

### DISCUSSION THREAD

**Discussion by jugalb-icpl** [DefectDojo/django-DefectDojo#11389] [Q&A] [answered]
Hi everyone,

Can anyone clarify how the Finding Group behaves during a test re-import?
Does it automatically create or delete finding groups based on changes?

Thanks!

**Comment by mtesauro** [comment, marked as answer]
Finding groups are just a collection of findings so reimport will handle each finding of a group as an individual finding updating it's status based on the comparison of the current vs the previous scan.

Reimport won't add or remove findings from a group - just change their status as I mentioned :point_up:  Humans (or API calls) are needed to manage Finding Groups.

**Reply by valentijnscholten** [reply]
@jugalb-icpl @mtesauro 
I believe reimport will add findings to a new or existing group if the user specifies a `group_by` in the reimport API request or UI dropdown. This `group_by` should be the same value is used on initial import to prevent undefined results :-)
